Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM ubuntu:focal
 
+# https://serverfault.com/questions/949991/how-to-install-tzdata-on-a-ubuntu-docker-image
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=Europe/Berlin
```

Comment:
I'd rather this was the system time zone or UTC, rather than a specific local timezone.

Reply:
Then ENV TZ line isn't need as UTC is used by default.

Reply:
thanks for the review. I have removed the ENV TZ and the build works fine